Documentation/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting started
+
+## Building the dex binary
+
+Dex requires a Go installation and a GOPATH configured. For setting up a Go workspace, refer to the [official documentation][go-setup]. Clone it down the correct place, and simply type `make` to compile the dex binary.
+
+```
+git clone https://github.com/coreos/dex.git $GOPATH/src/github.com/coreos/dex
+cd $GOPATH/src/github.com/coreos/dex
+make
+```
+
+A containerized version of this document using [rkt][rkt-repo] or Docker will be added later.
+
+## Configuration
+
+Dex is a single, scalable binary that pulls all configuration from a config file (no command line flags). Use the example config file defined in the `examples` folder to start up dex with an in-memory data store.
+
+```
+./bin/dex serve examples/config-dev.yaml
+```
+
+The [example config](../examples/config-dev.yaml) file documents many of the configuration options through inline comments. For extra config options, look at that file.
```

Comment:
This link point to https://github.com/coreos/dex/blob/readme-docs-v2/examples/config-dev.yaml. Is it meant to be https://github.com/coreos/dex/blob/dev/examples/config-dev.yaml?